src/api/posts/src/data/post.js

```diff
@@ -59,7 +60,7 @@ class Post {
     // Use the post's guid as our unique identifier
     this.id = hash(guid);
     this.title = title;
-    this.html = html;
+    this.html = linkifyHtml(html);
```

Comment:
We only need/want to do this if the post is from YouTube, I think.

Also, instead of doing it here, do it when we originally parse the content in the feed processor, and when the link is from YouTube (i.e., before we pass it to the `Post` constructor).